prog/vnet/update_firewall_rules.rb

```diff
@@ -258,8 +258,11 @@ def generate_globally_blocked_lists
     GloballyBlockedDnsname.each do |globally_blocked_dnsname|
       ips = globally_blocked_dnsname.ip_list || []
       ips.each do |ip|
-        globally_blocked_ipv4s << "#{ip}/32" if ip.ipv4?
-        globally_blocked_ipv6s << "#{ip}/128" if ip.ipv6?
+        if ip.version == 4
+          globally_blocked_ipv4s << "#{ip}/32"
+        else
+          globally_blocked_ipv6s << "#{ip}/128"
+        end
```

Comment:
another option
```suggestion
        globally_blocked_ipv4s << ip.version == 4 ? "#{ip}/32" : "#{ip}/128"
```

Reply:
I made the same mistake while I was working on this, but it actually appends to two separate arrays: `globally_blocked_ipv4s` and `globally_blocked_ipv6s`